packages/geocoder/src/index.story.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import getGeocoder from "./index";
+
+const GeocoderTester = ({
+  hereApiKey,
+  geocodeEarthApiKey,
+  onResults,
+  endpoint,
+  at
+}) => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [focusPoint, setFocusPoint] = useState({ lat: at.lat, lng: at.lng });
+  const [enableGeocodeEarth, setEnableGeocodeEarth] = useState(true);
+  const [enableHere, setEnableHere] = useState(true);
+
+  const geocoder = getGeocoder({
+    type: "HERE",
+    apiKey: hereApiKey,
+    focusPoint
+  });
+
+  const peliasGeocoder = getGeocoder({
+    type: "PELIAS",
+    apiKey: geocodeEarthApiKey,
+    baseUrl: "https://api.geocode.earth/v1",
+    size: 1,
+    focusPoint
+  });
+
+  const search = async () => {
+    const hereRes = enableHere
+      ? await geocoder[endpoint]({
+          text: searchTerm,
+          focusPoint,
+          point: focusPoint
+        })
+      : {};
+    const peliasRes = enableGeocodeEarth
+      ? await peliasGeocoder[endpoint]({
+          text: searchTerm,
+          focusPoint,
+          point: focusPoint
+        })
+      : {};
+    onResults({
+      hereRes,
+      peliasRes
```

Comment:
Just making sure, is the return value `{}` going to be correctly handled when otherwise a promise is returned?

Reply:
Yes, it's not a problem for Storybook.